crm_claim_code/models/crm_claim.py

```diff
@@ -29,3 +30,14 @@ def copy(self, default=None):
             default = {}
         default['code'] = self.env['ir.sequence'].get('crm.claim')
         return super(CrmClaim, self).copy(default)
+
+    @api.multi
+    @api.depends('code')
```

Comment:
Wow, I'm totally shocked now by this use that I haven't found, but it seems that both uses (depends and triggers) are interoperable in new and old API fields, so cool, let's keep it. Thanks for opening my eyes in this.